src/boot/measure.c

```diff
@@ -47,15 +50,18 @@ static int help(int argc, char *argv[], void *userdata) {
                "  status             Show current PCR values\n"
                "  calculate          Calculate expected PCR values\n"
```

Comment:
Maybe better to align descriptions for verbs?

Reply:
Actually, the #24351 PR adds more stuff to --help which makes this aligned properly again.